hazelcast/src/test/java/com/hazelcast/datalink/impl/DataLinkConsistencyCheckerTest.java

```diff
@@ -98,7 +98,7 @@ public void test_missingDataLinkWasAddedToDataLinkService() {
     @Test
     public void test_outdatedDataLinkWasAlteredInDataLinkService() {
         // given
-        linkService.replaceSqlDataLink(name, type, false, Collections.emptyMap());
+        linkService.replaceSqlDataLink(name, type, true, Collections.emptyMap());
```

Comment:
it would be good to have a test for case when only `shared` option is changed

Reply:
Actually, it was a point of this test. But, it's okay to have more.

Reply:
updated data link in this test has also different options. We should test if there are some datalink comparison conditions missing.